File: Leetcode/2501-3000/2579 - Count Total Number of Colored Cells.py
class Solution:
    def coloredCells(self, n: int) -> int:
        tiles = 0
        i = 1

        while i <= n:
            inc = 0
            if (i == 1):
                inc += 1
                tiles += inc
            else:
                inc += 4 * (i-1)
                tiles += inc
            i += 1
        return tiles

File: 2579 - Count Total Number of Colored Cells/2579 - Count Total Number of Colored Cells.cpp
class Solution {
public:
    long long coloredCells(int n) {
        long long tiles = 0;
        long long i = 1;

        while (i <= n) {
            long long inc = 0;
            if (i == 1) {
                inc++;
                tiles += inc;
            }
            else {
                inc += (4 * (i-1));
                tiles += inc;
            }
            i++;
        }
        return tiles;
    }
};

